File: SDKs/googleapis.com/privateca/v1beta1/typescript/src/sdk/models/shared/fetchcertificateauthoritycsrresponse.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// FetchCertificateAuthorityCsrResponse
/** 
 * Response message for CertificateAuthorityService.FetchCertificateAuthorityCsr.
**/
export class FetchCertificateAuthorityCsrResponse extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=pemCsr" })
  pemCsr?: string;
}

File: SDKs/googleapis.com/privateca/v1beta1/go/pkg/models/shared/fetchcertificateauthoritycsrresponse.go
package shared

// FetchCertificateAuthorityCsrResponse
// Response message for CertificateAuthorityService.FetchCertificateAuthorityCsr.
type FetchCertificateAuthorityCsrResponse struct {
	PemCsr *string `json:"pemCsr,omitempty"`
}
